torch/_decomp/decompositions.py

```diff
@@ -301,8 +301,8 @@ def _prelu_kernel_backward(
 def rrelu_with_noise(
     self: Tensor,
     noise: Tensor,
-    lower: float,
-    upper: float,
+    lower: float = 0.125,
+    upper: float = 0.3333333333333333,
```

Comment:
how did you come up with these numbers?

Reply:
Probably here? https://github.com/pytorch/pytorch/blob/71343507cda63d0c1c38fde6a7f6d3a4e5182588/torch/csrc/utils/schema_info.cpp#L291

Reply:
Actually from here - but those same numbers are used with rrelu() and rrelu_with_noise() all over the place.

https://github.com/pytorch/pytorch/blob/71343507cda63d0c1c38fde6a7f6d3a4e5182588/torch/csrc/utils/schema_info.cpp#L291

Reply:
Lol - just realized my previous comment had the same file.  But still - those same numbers are used for rrelu() and rrelu_with_noise() all over the place.